.eslintrc.json

```diff
@@ -24,6 +24,8 @@
     "no-restricted-exports": "off",
     "react/jsx-props-no-spreading": "off",
     "import/no-cycle": "off",
-    "import/prefer-default-export": "off"
+    "import/prefer-default-export": "off",
+    "jsx-a11y/label-has-for": 0
```

Comment:
Big no here, we don't add eslint exceptions, we obey the rules that exists. Maybe try adding an "htmlFor" attribute to your label, that might fix the error that you will be getting.